runtime/ztests/expr/function/length.yaml

```
@@ -1,9 +1,11 @@
-spq: SELECT LENGTH('W3Schools.com');
+spq: SELECT LENGTH(s);
```

Nit: No need for records here. Just make the input a sequence of strings.

```suggestion
spq: SELECT LENGTH(this);
```

---

You get a semantic error if you pass `this` into this expression

---

```
super compile -dag 'SELECT LENGTH(this);'
cannot reference 'this' in relational context; consider the 'yield' operator at line 1, column 15:
SELECT LENGTH(this);
              ~~~~
```